src/models/StyledComponent.js

```diff
@@ -110,7 +87,8 @@ class BaseStyledComponent extends Component<*, BaseState> {
 
   generateAndInjectStyles(theme: any, props: any) {
     const { attrs, componentStyle, warnTooManyClasses } = this.constructor
-    const styleSheet = this.context[CONTEXT_KEY] || StyleSheet.master
+    // const styleSheet = this.context[CONTEXT_KEY] || StyleSheet.master
+    const styleSheet = StyleSheet.master
```

Comment:
I'm unsure what this previously was for, but I think it will be fine just assigning it to `Stylesheet.master`?

Reply:
@kitten can you comment on this? I think it's okay

Reply:
Oh just realized that that context comes from the `StyleSheetManager`, so I'll have to wrap it in a `StyleSheetConsumer` or similar to get the correct value